content/en/tutorials/cloud-integration-testing-localStack-ci/index.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "Cloud Integration Testing with LocalStack & GitHub Action"
+linkTitle: "Cloud Integration Testing with LocalStack & GitHub Action"
+weight: 11
+description: >
+  We'll explore how to leverage LocalStack, a drop-in replacement for AWS and GitHub Actions, a CI/CD tool, to streamline our cloud integration testing process. This reduces cloud reliance, cuts costs, and significantly speeds up CI testing, improving overall developer productivity.
+type: tutorials
+teaser: ""
+teaser: ""
```

Comment:
Why are there two teaser fields? I would also suggest to link the apt teaser here